scripts/redshift/loader.py

```diff
@@ -2,11 +2,59 @@
 import concurrent.futures
 import multiprocessing
 import os
+from enum import Enum
 
 from matrix.common import etl
 from matrix.common.aws.redshift_handler import RedshiftHandler
 from matrix.common.query_constructor import format_str_list
 
+
+class MetadataSchemaName(Enum):
+    PROJECT = "project"
+    LIBRARY_PREPARATION_PROTOCOL = "library_preparation_protocol"
+    ANALYSIS_PROTOCOL = "analysis_protocol"
+    SPECIMEN_FROM_ORGANISM = "specimen_from_organism"
+    DONOR_ORGANISM = "donor_organism"
+    CELL_LINE = "cell_line"
+    CELL_SUSPENSION = "cell_suspension"
+    ORGANOID = "organoid"
+
+
+LATEST_SUPPORTED_MD_SCHEMA_VERSIONS = {
+    MetadataSchemaName.PROJECT: {
+        'major': 14,
+        'minor': 0
+    },
+    MetadataSchemaName.LIBRARY_PREPARATION_PROTOCOL: {
+        'major': 6,
+        'minor': 1
+    },
+    MetadataSchemaName.ANALYSIS_PROTOCOL: {
+        'major': 9,
+        'minor': 0
+    },
+    MetadataSchemaName.SPECIMEN_FROM_ORGANISM: {
```

Comment:
We also have minimum supported versions. For example, for `specimen_from_organism`, it's 9.0.0.

Reply:
Gotcha. Do you have a list of minimum supported versions for all schemas?

Reply:
Talked to @maniarathi about this on slack. The answer is that I do not have a list, and it's pretty tedious to create one. But, the minimum for `specimen` is probably the only relevant one.